claragenomics/dl4atac/losses.py

```diff
@@ -75,7 +75,7 @@ def mse(self):
         Mean squared error loss type.
     
         Return:
-            loss_func:
+            Torch MSE Loss function.
```

Comment:
Should it be `loss_func : Torch MSE Loss function`

Reply:
We've been following the Google docstring style in other projects. return only requires a description in that format

https://stackoverflow.com/questions/3898572/what-is-the-standard-python-docstring-format

Reply:
What about multiple variable return ?

Reply:
That would be described as 
```
tuple with first element as X, second element as Y, third element as Z, etc.
```